app/build.gradle

```diff
@@ -133,6 +133,10 @@ dependencies {
     androidTestImplementation "androidx.test:rules:1.4.0"
     androidTestImplementation "androidx.test.espresso:espresso-core:3.4.0"
     implementation 'com.github.wseemann:FFmpegMediaMetadataRetriever:1.0.14'
+
+    implementation 'org.apache.tika:tika-core:2.4.0'
+    //implementation 'org.apache.tika:tika-parsers-standard-package:2.4.0'
```

Comment:
What `tika-parsers-standard-package` could be useful for?

Reply:
Not needed for now, I have deleted it.